manim/mobject/geometry.py

```diff
@@ -767,6 +767,30 @@ def __init__(self, outer_radius=1, inner_radius=0, **kwargs):
 
 
 class Annulus(Circle):
+    """Region between two concentric :class:`Circle`s.
+
+    Parameters
+    ----------
```

Comment:
Make sure to add the typings in the function and not the documentation part.
See: https://docs.manim.community/en/latest/contributing/typings.html

Reply:
I'm sorry I don't understand. I just did what I saw in other examples

Reply:
You are supposed to remove inner_radius ~: :class:`float`, optional~ 
and put it under the function parameters like this:
`inner_radius: Optional[float] = 1,`

Go through the link once again to get an idea on what typings to use where.